crates/fuel-core/src/database.rs

```diff
@@ -278,8 +285,15 @@ where
         }
         #[cfg(feature = "rocksdb")]
         {
-            Self::rocksdb_temp(StateRewindPolicy::NoRewind)
-                .expect("Failed to create a temporary database")
+            Self::rocksdb_temp(
+                StateRewindPolicy::NoRewind,
+                DatabaseConfig {
+                    cache_capacity: None,
+                    max_fds: 512,
```

Comment:
why is it set to `512` here and `-1` in other places? perhaps this can be -1 too?

Reply:
In benches we want to use full capacity. But in the real node running we want to have a decent usage of the machine by the db.

Reply:
cool